app/routes/apply/new-eligibility/benefits.js

```diff
@@ -3,9 +3,19 @@ const UrlPathValidator = require('../../../services/validators/url-path-validato
 const ValidationError = require('../../../services/errors/validation-error')
 const claimTypeEnum = require('../../../constants/claim-type-enum')
 
+const REFERENCE_DOB_ERROR = '?error=expired'
+
 module.exports = function (router) {
   router.get('/apply/:claimType/new-eligibility/:dob/:relationship', function (req, res) {
```

Comment:
dob/relationship should be in session and not in path for both GET and POST in this route